CHANGELOG.md

```diff
@@ -40,6 +40,8 @@
 
 ##### Bug Fixes
 
+* `CXComment.commandName()` was returning nil on custom code comments since xcode 8.1 sdk. This caused a force unwrap when generating documentation. Inline command comment is now used as a fallback to catch this edge case.
```

Comment:
Also, we hard-wrap CHANGELOG lines at 80 characters. See [CONTRIBUTING.md](https://github.com/jpsim/SourceKitten/blob/master/CONTRIBUTING.md).